src/problems/Albany_ResponseUtilities.hpp

```diff
@@ -38,18 +38,16 @@ class ResponseUtilities
   constructResponses(
     PHX::FieldManager<PHAL::AlbanyTraits>& fm0,
     Teuchos::ParameterList& responseList,
-    Teuchos::RCP<Teuchos::ParameterList> paramsFromProblem,
```

Comment:
The state mgr was never used, so again, I decided to change the method signature, to keep things slimmer.